cli/src/agent.rs

```diff
@@ -0,0 +1,144 @@
+use crate::{startup::get_executor_url, util::query};
+use anyhow::{anyhow, Context, Result};
+use graphql_client::{GraphQLQuery, Response};
+
+#[derive(GraphQLQuery)]
+#[graphql(
+    schema_path = "schema.gql",
+    query_path = "src/agent.gql",
+    response_derives = "Debug"
+)]
+pub struct RequestCapability;
+
+pub async fn run_request_capability() -> Result<String> {
+    let query = RequestCapability::build_query(request_capability::Variables {
+        app_name: "AD4M cli".to_string(),
```

Comment:
Is this mock data expected?

Reply:
That's not mock data, but the name of this app.